vivisect/__init__.py

```diff
@@ -739,13 +739,29 @@ def getExports(self):
         """
         return list(self.exports)
 
-    def addExport(self, va, etype, name, filename):
+    def addExport(self, va, etype, name, filename, makeuniq=False):
```

Comment:
We'll need to make makeuniq=True to avoid breaking everybody who already uses this API.

Reply:
actually, the current API is synonymous with makeuniq=False (which is why i chose that)
(if not makeuniq, do the thing we did before)

Reply:
are you satisfied with this answer?  we made the default be the same behavior as was before.  "makeuniq" means, create a name if one already exists.  if not, just fail if the name exists.